assets/src/edit-story/components/dropTargets/karma/order.karma.js

```diff
@@ -54,14 +59,13 @@ describe('Drop-Target order', () => {
     const replacementImageFrame = fixture.editor.canvas.framesLayer.frame(
       replacementImage.id
     ).node;
-
     await fixture.events.mouse.seq(({ moveRel, moveBy, down, up }) => [
       moveRel(replacementImageFrame, 10, 10),
       down(),
       moveBy(0, -20),
       up(),
     ]);
-
+    await waitForElementToBeRemoved(replacementImageFrame);
```

Comment:
This is added because? Flaky tests or?

Reply:
Yes, to ensure waiting until the original image has been removed and the dropping has happened.